TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TestOpModes/DriveTrainTeleOpTest.java

```diff
@@ -0,0 +1,90 @@
+package org.firstinspires.ftc.teamcode.TestOpModes;
+
+import com.qualcomm.hardware.kauailabs.NavxMicroNavigationSensor;
+import com.kauailabs.navx.ftc.AHRS;
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.IntegratingGyroscope;
+
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+import org.firstinspires.ftc.teamcode.DriveTrain;
+
+/**
+ * Created by FTC 4316 on 10/29/2017
+ */
+@TeleOp(name="Drive Train TeleOp test", group="Test")
+public class DriveTrainTeleOpTest extends OpMode {
+
+    private DriveTrain driveTrain;
+
+    private double x;
+    private double y;
+    private double z;
+
+    DcMotor leftFront;
+    DcMotor rightFront;
+    DcMotor leftRear;
+    DcMotor rightRear;
+
+    private double power;
+    private double finches;
+
+    private boolean isFinished;
+
+    NavxMicroNavigationSensor gyroScope;
+    IntegratingGyroscope gyro;
+
+    @Override
+    public void init() {
+
+
+
+        leftFront = hardwareMap.dcMotor.get("leftfront");
+        rightFront = hardwareMap.dcMotor.get("rightfront");
+        leftRear = hardwareMap.dcMotor.get("leftrear");
+        rightRear = hardwareMap.dcMotor.get("rightrear");
+
+        power = 0.1;
+        finches = 5;
+
+        gyroScope = hardwareMap.get(NavxMicroNavigationSensor.class, "navx");
+        gyro = (IntegratingGyroscope)gyroScope;
```

Comment:
#fix change `gyroScope` to something a little more descriptive. Maybe `navxGyroReference`?